### ISSUE TITLE
[Accessibility][Reflow -Flow Chart]: At 400%, the flow chart is not visible properly.

### ISSUE THREAD

**Issue by ashish315** [microsoft/BotFramework-Composer#2053]
**User Experience** 
Users having low vision will get difficulties to interact with the chart available on the page as the charts  are not visible properly in 400% zoom.

**Test Environment**
Browser: Microsoft Edge Canary Version: 82.0.425.0 (Official build)
OS build: 2004 (19564.1005)
Screen Reader: Narrator
URL: [Bot Framework Composer](http://localhost:3000/dialogs/Main)
[**Prerequisite Here**](https://github.com/microsoft/BotFramework-Composer)

**Repro Steps:** 
1. Open the above mentioned URL in EDGE and set the zoom at 400%.
2. Select 'Design Flow' from left navigation.
3. Now, navigate to the flow chart.
4. Observe the issue in the flow chart area.

**Actual:** 
When the zoom is at 400%, the chart is not visible properly. As well as no vertical slide bar available to scroll down the chart.

**Note:**
This issue is repro throughout the application.

**Expected:**
The chart should be clearly visible as well as their should a vertical slide bar so that the user will able to observe and understand the flow of the chart.

**MAS Impacted:** [MAS1.4.10](https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/Doc.aspx?sourcedoc=%7BE12828DF-B35C-4F4D-9289-1F932E19F8CF%7D&file=MAS%201.4.10%20%E2%80%93%20Reflow.docx&action=default&mobileredirect=true&cid=2da5c2f2-146e-4e15-a93a-e961eb2c1cd2)

**Attachment:**
![MAS1 4 10_flow chart at 400%](https://user-images.githubusercontent.com/60913131/75111172-ae54e280-565c-11ea-8d1b-d4fdd540266f.png)
![At 400% property pane is not visible](https://user-images.githubusercontent.com/60913131/75511712-0309ac00-5a15-11ea-9fa4-5a071f503d88.png)

**Comment by DesignPolice** [comment]
@cwhitten how did PVA handle this, since we are using their canvas... @ashish315  at 400% navigating with "Tabs" would you expect each tab to center as you go through each list or object? Do you have any examples of things you have seen that make sense to you?

**Comment by ashish315** [comment]
@DesignPolice You can refer Office.com for the reference as well as i attached a video for your reference. And getting the control in center is not our priority, getting the control clearly visible with all the contents are our priority.
[400% compare with OCOM.zip](https://github.com/microsoft/BotFramework-Composer/files/4265340/400.compare.with.OCOM.zip)

**Comment by DesignPolice** [comment]
@a-b-r-o-w-n take a look at the video on this one, see if that aligns to what you were thinking for this fix? 
It is a little easier than I thought, the only trick will be jumping to the nodes without a pause on the visual canvas since it will be too big to focus.

**Comment by a-b-r-o-w-n** [comment]
This is not what I was suggesting. I suggested a "mobile" like layout. That will alleviate some of the page real estate issues, but there may be more work required to ensure the focus is properly managed.

**Comment by DesignPolice** [comment]
okay. Well maybe some of both of those things would be helpful... for me visually just having things center up as much as we can would be great. The Composer icon, as much as I like it seems to get really large with the 400% zoom - 

Attached

Lets see what we can do. 

![Screen Shot 2020-03-03 at 10 35 12 AM](https://user-images.githubusercontent.com/45223620/75807745-c6062800-5d3a-11ea-8559-8ca0ab55b609.png)

**Comment by alanlong9278** [comment]
Have fixed the composer icon display error in #2608 . And the focus control logic in `visual editor` will be refactor soon.

**Comment by ashish315** [comment]
As checked the issue is resolved. So, HCL is closing the issue.
